library/src/main/java/com/chuckerteam/chucker/api/internal/ui/transaction/TransactionActivity.java

```diff
@@ -148,32 +147,46 @@ private void share(String content) {
         startActivity(Intent.createChooser(sendIntent, null));
     }
 
-    static class Adapter extends FragmentPagerAdapter {
-        final List<TransactionFragment> fragments = new ArrayList<>();
-        private final List<String> fragmentTitles = new ArrayList<>();
+    static class Adapter extends FragmentStatePagerAdapter {
```

Comment:
Can I ask you to have this class as non static?
Moreover `PagerAdapter` is probably a better name